chains/73771/0x26b77eeF7A38E3FD8C631FF8a268a5BB98CE1552/contract-information.json

```diff
@@ -33,5 +33,5 @@
             "coingecko": "https://www.coingecko.com/en/coins/treasure-under-sea"
         }
     ],
-    "logoUri": "https://raw.githubusercontent.com/ava-labs/subnet-assets/main/chains/73771/0x26b77eeF7A38E3FD8C631FF8a268a5BB98CE1552/92fd487198904d13bf1cec58111bed39.png"
+    "logoUri": "https://raw.githubusercontent.com/ava-labs/subnet-assets/main/chains/73771/0x26b77eeF7A38E3FD8C631FF8a268a5BB98CE1552/logo.png"
```

Comment:
We don't actually need to store the `logoUri` here the image file name is deterministic

Reply:
I think we should keep it, since there are some contracts that still have external images on the mainnet chain